### DISCUSSION TITLE
Fetching external api with post request fron vercel

### DISCUSSION THREAD

**Discussion by vanyapr** [vercel/next.js#23331] [Help]
I use vercel to run the app next.js. How can I access the external api endpoint? I'm trying to use a node-fetch module like this: fetch (apiUrlAddress, {method: "post”, ...data}), everything works on the local host, but when you deploy to vercel, calls to the api stop working. Any ideas?

**Comment by akellbl4** [comment]
Polly for fetch on server side already [bundled in Next.js](https://nextjs.org/docs/basic-features/supported-browsers-features#server-side-polyfills). You can use just `fetch`.
Do you have errors in Vercel's log?

**Reply by vanyapr** [reply]
No, dont have any errors in app consol. I deployed app on my own cloud server and it works fine atm. But on the vercel external urls fetch still does not works.

**Reply by akellbl4** [reply]
By the way, I haven't experienced problems with fetching external resources on Vercel
If your application is working I think the problem resolved at this point ;)

**Reply by vanyapr** [reply]
My problem is when user submit a data, my app should send this date to external api via post fetch. This does not happens: app recieves data and does nothing. Just using my own cloud instead of vercel, because cercel does not works at all.
